common/source/docs/common-frsky-flightdeck.rst

```diff
@@ -0,0 +1,43 @@
+.. _common-frsky-flightdeck:
```

Comment:
This is a product page and should be at the vendors page. The wiki should only reflect ArduPilot's capabilities.